_docs/hackathon-preperation-checklist.md

```diff
@@ -0,0 +1,52 @@
+The UWB Hacks is a 13-hour event, which can be a little daunting. Not sure how to prepare? Don't worry, this list has you covered.
+
+# 1-2 Weeks Before
+
+- Make a team
+    - 6 should be the max, but any group size below that is fine.
+- Brainstorm ideas
+    - Not sure where to start? Check out our [project ideas page](_docs/project-ideas.md) to get your creative juices flowing.
```

Comment:
Use the HTML permalink for internal linking; I figured out earlier this week that linking to the `.md` file will cause a 404 error, because all the markdown docs are converted by Jekyll to HTML pages for us, and the pages are generated according to the permalinks we assign them.
```suggestion
    - Not sure where to start? Check out our [project ideas page](project_ideas.html) to get your creative juices flowing.
```